doc/source/examples/incompressible-flow/2d-transient-around-ahmed-body/2d-transient-around-ahmed-body.rst

```diff
@@ -0,0 +1,214 @@
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 1; 0; 0
+        end
+    end
+
+    subsection boundary conditions
+        set number                  = 3
+        subsection bc 0
+            set type              = noslip
+        end
+        subsection bc 1
+            set type              = function
+        subsection u
+            set Function expression = 1
+        end
+        subsection v
+            set Function expression = 0
+        end
+        subsection w
+            set Function expression = 0
+        end
+    end
+        subsection bc 2
+            set type              = slip
+        end
+    end
+
+Simulation control
+~~~~~~~~~~~~~~~~~~
+Time integration is defined by a 1st order backward differentiation (``bdf1``), for a 4 seconds simulation (``time end``) with a 0.01 second ``time step``. The ``output path`` is defined to save obtained results in a sub-directory, as stated in `Simulation Control <https://lethe-cfd.github.io/lethe/parameters/cfd/simulation_control.html>`_:
+
+.. code-block:: text
+
+    subsection simulation control
+        set method                  = bdf1
+        set output frequency        = 1
+        set output name             = ahmed-output
+        set output path             = ./Re720/
+        set time end                = 4
+        set time step		      = 0.01
+    end
+
+
+.. Warning::
+   To successfully launch the simulation, the ``output path`` where the results are saved (in this example, the folder ``Re720``) must already exist. Otherwise, the simulation will hang because it will be unable to save the results.
+
+Ahmed bodies are typically studied considering a 60 m/s flow of air. Here, the flow speed is set to 1 (``u = 1``) so that the Reynolds number for the simulation (``Re = uL/ν``, with ``L`` the height of the Ahmed body) is varied  by changing the ``kinematic viscosity``:
+
+.. code-block:: text 
+
+    subsection physical properties
+       subsection fluid 0
+           set kinematic viscosity         = 4e-4
+        end
+    end
+
+The simulation is launched in the same folder as the ``.prm`` and ``.msh`` file, using the ``gls_navier_stokes_2d`` solver. To decrease simulation time, it is advised to run on multiple cpu, using ``mpirun``:
+
+To do so, copy and paste the ``gls_navier_stokes_2d`` executable to the same folder as your ``.prm`` file and launch it running the following line:
+
+.. code-block:: text
+
+      mpirun -np 6 gls_navier_stokes_2d ahmed.prm
+
+where 6 is the number of cpu used. The estimated execution time for a 4 seconds simulation is 6 minutes and 53 seconds.
```

Comment:
If you want to compare with 1 cpu (and demonstrate that mpirun is fun), the system time of the simulation with 1 cpu took 30 minutes and 37 seconds. :)